vnext/Desktop/WebSocket.h

```diff
@@ -89,15 +90,25 @@ class BaseWebSocket : public IWebSocket
   #pragma endregion
 };
 
-template<typename Protocol, typename Socket = boost::asio::basic_stream_socket<Protocol>, typename Resolver = boost::asio::ip::basic_resolver<Protocol>>
-class WebSocket : public BaseWebSocket<Protocol, Socket, Resolver>
+class WebSocket
+  : public  BaseWebSocket
```

Comment:
What's the benefit of removing the C++ template

Reply:
1. More accurate representation of the intent of the class. You don't add type parameters to an instance of `WebSocket`. `WebSocket` is a specialization of `BaseWebSocket`.
1. Less boilerplate code (especially in `WebSocket.cpp`.